CONTRIBUTING.rst

```diff
@@ -0,0 +1,61 @@
+How to Contribute
+=================
+
+We can use contributions to the images here, but can also use your help in
+testing the images. If you'd like to sign up for beta testing the ``latest``
+image, sign up here:
+
+https://goo.gl/forms/3oAbxkGMUiEZz2782
+
+We'll need to verify that you are the account holder, and that you made the
+request for the projects that you listed.
+
+Testing
+-------
+
+You can test your image by running a shell in a container::
+
+    docker run --rm -t -i readthedocs/build:latest /bin/bash
+
+This will put you into the root path in the container, as the ``docs`` user.
+From here you can head to your home path (``cd ~docs``) and run normal
+Python/Sphinx/etc operations to see if your changes have worked.
+
+Releases
+--------
+
+Theses images are all built from our `automated Docker Hub repository`_. Here is
+how the images are currently built:
+
+build:1.0
+    From the ``1.0`` tag. These tags should only represent commits to the
+    ``releases/1.x`` branch.
+
+build:2.0
+    From any tag matching ``2.0[0-9.]*``. These tags should only represent
+    commits to the ``releases/2.x`` branch.
+
+build:2.1
```

Comment:
I should clarify, we don't have a `2.1` yet. Docker hub is set up to build it however.